integration/kubernetes/init.sh

```diff
@@ -74,6 +74,15 @@ fi
 
 echo "Start ${cri_runtime} service"
 sudo systemctl start ${cri_runtime}
+for i in {1..5}; do
+       sleep 5
+       if [ -f "${cri_runtime_socket}" ]; then
```

Comment:
I see some other white space cleanup, as will as this retry. I’m okay w the whitepace being included in the commit, but what’s this for?

Reply:
Because I  the cri-containerd job is running k8s two times, the contaienrd service is restarted and seems that kubeadm tries to connect and is not ready yet. I add a some retries to check the socket exist.